ci/run_autoscaling_osp18.yml

```diff
@@ -1,35 +1,25 @@
 ---
-- name: "Run functional test playbooks"
+- name: Run telemetry autoscaling tests on osp18
   hosts:  "{{ cifmw_target_hook_host | default('localhost')  }}"
   gather_facts: true
   environment:
     KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
     PATH: "{{ cifmw_path }}"
   vars_files:
     - vars/common.yml
+    - vars/osp18_env.yml
   tasks:
-    - name: "Create log dir"
-      ansible.builtin.file:
-        path: "{{ logs_dir }}"
-        state: directory
-        mode: "0755"
+    - block:
+        - name: "Create log dir"
+          ansible.builtin.file:
+            path: "{{ logs_dir }}/autoscaling"
+            state: directory
+            mode: "0755"
```

Comment:
With a follow-up to update the logs location, we can get the plugin to make sure the dir is created.